tests/shakedown/shakedown/clients/authentication.py

```diff
@@ -73,24 +74,16 @@ def dcos_acs_token():
     logger.info('Authenticating with DC/OS cluster...')
 
     # Try token from dcos cli session
-    # try:
-    #     clusters = run_dcos_command('cluster list --attached --json')
-    #     clusters = json.loads(clusters)
-    #     clusters_by_url = {c['url']: c['cluster_id'] for c in clusters}
-    #     cluster_id = clusters_by_url.get(dcos_url())
-    #     if cluster_id is not None:
-    #         dcos_cli_file = path.expanduser('~/.dcos/clusters/{}/dcos.toml'.format(cluster_id))
-    #         dcos_cli_config = toml.load(dcos_cli_file)
-    #         token = dcos_cli_config['core']['dcos_acs_token']
-
-    #         # TODO: Use token to ping leader and verify that it's valid.
-
-    #         logger.info('Authentication using DC/OS CLI session ✓')
-    #         return token
-    #     else:
-    #         logger.warning('Authentication using DC/OS CLI session ✕')
-    # except Exception:
-    #     logger.warning('Authentication using DC/OS CLI session ✕')
+    try:
+        (token, _, _) = run_dcos_command('config show core.dcos_acs_token', print_output=False)
+        token = token.rstrip()
+
+        url = dcos_url_path('service/marathon/ping')
+        requests.get(url, auth=DCOSAcsAuth(token), verify=False).raise_for_status()
```

Comment:
Would it make sense to add this to the other methods for retrieving the token too?

Reply:
It's a bit of a nit, but this could also fail for other reasons (Marathon not yet being able to service requests) that are unrelated to the token being invalid.

Reply:
Could be. This is my poor man's way of verify that the DC/OS CLI session is valid. Should we retry?

Reply:
✅